templates/operation.tmpl

```diff
@@ -35,4 +35,42 @@ type {{.ReceiverName}} {{template "schema.tmpl" .Items}}
       {{- end}}
     {{- end}}
   {{- end}}
+{{- end}}
+
+{{range .Handlers -}}
+type Impl{{.Name}} struct{}
```

Comment:
These implementations should actually go in a separate file (that we won't overwrite)